### ISSUE TITLE
Incorrectly positioned help text on error in horizontal form layout

### ISSUE THREAD

**Issue by gameiqinc** [FriendsOfCake/bootstrap-ui#36]
The help block for error messages should be within the input column div tag when generated for horizontal forms.

The currently generated code looks like this:
```html
<div class="form-group required has-error">
	<label class="control-label col-md-2" for="field">Field</label>
	<div class="col-md-10">
		<div class="input-group">
			<input type="number" name="field" required="required" id="field" class="form-control " value="">
		</div>
	</div>
	<p class="help-block">Error message</p>
</div>
```

It should be like this:
```html
<div class="form-group required has-error">
	<label class="control-label col-md-2" for="field">Field</label>
	<div class="col-md-10">
		<div class="input-group">
			<input type="number" name="field" required="required" id="field" class="form-control " value="">
		</div>
		<p class="help-block">Error message</p>
	</div>
</div>
```

**Comment by jadb** [comment]
Thanks! I have put this for 0.3 which I am working on. I am not sure it exists on that branch, so will have to check by using your markup for a test case.

**Comment by gameiqinc** [comment]
NP! Good job with this plugin, btw. Very useful.

**Comment by ADmad** [comment]
Fixed on branch 0.3
